k8s/dialer_test.go

```diff
@@ -98,6 +100,8 @@ func TestDialInClusterService(t *testing.T) {
 	defer cliSet.CoreV1().Services(testingNS.Name).Delete(ctx, nginxService.Name, deleteOpts)
 	t.Log("created svc: ", nginxService.Name)
 
+	time.Sleep(time.Second * 10)
```

Comment:
Probably could use a bit of context on why this wait is here, like `waiting for nginx service to be ready`